tests/acceptance/welcome/payment-info-test.js

```diff
@@ -13,14 +13,25 @@ function visitPaymentInfoWithApp(options, userData){
 
   signInAndVisit(url, userData);
   andThen(function(){
-    if (!options) { return clickButton('Skip this step'); }
+    if (!options) {
+      return clickButton('Skip this step');
+    }
 
-    if (options.dbType){ click(`.${options.dbType} a`); }
+    if (options.dbType) {
+      click(`.${options.dbType} a`);
+    }
+    if (options.dbInitialDiskSize) {
+      andThen(function(){
+        Ember.$('.slider').trigger('slide', options.dbInitialDiskSize);
```

Comment:
`triggerSlider` will do. I was pained b/c there is also a `set` trigger. It gave me pause from making a `set` helper.

Will make a `triggerSlider`